shared/styles.css

```
@@ -0,0 +1,606 @@
+  --error: #dc2626;
+  --error-bg: #fef2f2;
+  --radius: 12px;
+  --radius-sm: 8px;
+  --shadow: 0 1px 3px rgba(0,0,0,0.08);
+}
+
+/* Reset */
+*, *::before, *::after { box-sizing: border-box; margin: 0; padding: 0; }
+
+body {
+  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, sans-serif;
```

Stylelint (17.4.0)

[error] Unexpected quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)